src/app/home-page/components/task-list/task-list.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="container">
-  <div class="d-flex justify-content-between">
+  <div *ngIf="enableTaskManaging" class="d-flex justify-content-between">
```

Comment:
there was a comment from Anatolyi: as for me - this approach is okay, it will help with duplicating the task-list HTML and makes this component more re-usable. Or maybe I didn't understand what exactly Anatolyi meant